### DISCUSSION TITLE
AVATAR_DIRについて

### DISCUSSION THREAD

**Discussion by REI-MORI** [sinProject-Inc/talk#1447] [Q&A]
以下のファイルで `import { AVATAR_DIR } from '$env/static/private'` という記述がされていますが、以下のエラーが発生します。

AVATAR_DIRをimportしているファイル

- src/lib/avatar/avatar_file_path.ts
- src/routes/(authed)/profile/+page.server.ts

エラー
`Error: Module '"$env/static/private"' has no exported member 'AVATAR_DIR'.`

.svelte-kit/ambient.d.ts の中にAVATAR_DIRが定義されていないのが原因なのですが、.svelte-kitは.gitignoreに定義されており、修正を行ってもGitに認識されません。.gitignoreから.svelte-kitを削除し、.svelte-kit/ambient.d.tsにAVATAR_DIRを定義してもよいでしょうか。

**Comment by sinproject-iwasaki** [comment]
@REI-MORI 

こちらで anbient.d.ts から AVATAR_DIR の定義を慶して確認しました。

1. .env に AVATAR_DIR の定義を作成する
2. `npm run dev` を実行する

これで、anbient.d.ts が更新され、参照エラーが無くなる。

と思われます。一度お試しくださいませ。

**Reply by REI-MORI** [reply]
@sinproject-iwasaki ありがとうございます。手順通りに行ったら参照エラーがなくなりました。
